src/LayerUtil/LayerUtil.js

```diff
@@ -67,6 +69,46 @@ class LayerUtil {
     return /** @type {[number, number, number, number]} */ (extent);
   }
 
+  /**
+   * Returns all attributions as text joined by a separator.
+   *
+   * @param {import("ol/layer/Layer").default} layer The layer to get the attributions from.
+   * @param {string} separator The separator separating multiple attributions.
+   * @returns {string} The attributions.
+   */
+  static getLayerAttributionsText = (layer, separator = ', ') => {
+    const attributionsFn = layer.getSource()?.getAttributions();
+    // @ts-ignore
+    const attributions = attributionsFn ? attributionsFn(undefined) : null;
+
+    let attributionString;
+    if (Array.isArray(attributions)) {
+      attributionString = attributions.map(LayerUtil.getTextFromHtml).join(separator);
+    } else {
+      attributionString = attributions ? LayerUtil.getTextFromHtml(attributions) : '';
+    }
+    return attributionString;
+  };
+
+
+  /**
+   * Converts a html string into text using DOMParser.
+   *
+   * Credits: https://stackoverflow.com/questions/822452/strip-html-from-text-javascript/47140708#47140708.
+   *
+   * @param {string} html The html to convert.
+   * @returns {string} The output text. Returns an empty string if parsing fails.
+   */
+  static getTextFromHtml = (html) => {
```

Comment:
Not really blocking, but as this would become part of the API one might wonder why this is located in the `LayerUtil`. Probably better placed in the `base-util` (or an internal helper class inside the `ol-util`)?

Reply:
I just found StringUtil.stripHTMLTags() in base-util, which does the same thing :tada: 

fixed here 055d94c89784d7c621afd3d3d2825b835db9b5de